extensions/amp-story/1.0/amp-story.css

```diff
@@ -433,6 +433,10 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
                        "lower-third" !important;
 }
 
+amp-story-grid-layer.i-amphtml-has-quiz {
+  align-content: normal;
```

Comment:
Why is this needed? What happens if publishers override it?
Is there anything else we could do to avoid changing the default `align-content` value? Changing it for only one `amp-story-grid-layer` will be very confusing to publishers